documentation/assemblies/managing/assembly-rolling-updates.adoc

```diff
@@ -0,0 +1,29 @@
+// This assembly is included in the following assemblies:
+//
+// using/assembly-management-tasks.adoc
+
+[id='assembly-rolling-updates-{context}']
+= Performing manual rolling updates of Kafka and ZooKeeper clusters
+
+Strimzi supports the use of annotations on `StatefulSet` and `Pods` resources to manually trigger a rolling update of Kafka and ZooKeeper clusters through the Cluster Operator.
```

Comment:
For some reason, _`Pods` resources_ sounds weird. Can it be just _`Pod` resources_ with Pod in singular?